thirdparty_containers/jenkins/playlist.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>jenkins_test</testCaseName>
+	<command>docker run --rm -v $(JDK_HOME):/java  adoptopenjdk-jenkins-test:latest "jvm pos neg"; \
+		$(TEST_STATUS)</command>
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+		</subsets>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>external</group>
+		</groups>
+	</test>
+</playlist>
```

Comment:
Please add a blank line at the end of the file.

Reply:
What is the purpose of the blank line, aesthetics or necessary?

Reply:
We generally add a newline at the end of every file because some tools might misbehave or consider the file to be truncated if the last line of the file is not a blank line. Some more explanations behind this practice: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
Added new line at EOF.